app/models/float_field_type.rb

```diff
@@ -0,0 +1,39 @@
+class FloatFieldType < FieldType
+  attr_accessor :float
+ 
+  validates_numericality_of :float
+  validate :less_than, if: Proc.new { |int| validate_key(:max) }
+  validate :greater_than, if: Proc.new { |int| validate_key(:min) }
```

Comment:
I know this literally doesn't matter code-wise, but could you please change the `int` to `float` since the field type has changed